src/view/components/formikReactSpectrum3/formikComboBox.jsx

```diff
@@ -15,7 +15,21 @@ import PropTypes from "prop-types";
 import { ComboBox } from "@adobe/react-spectrum";
 import { useField } from "formik";
 
-const FormikComboBox = ({ name, width, ...otherProps }) => {
+/**
+ * @param {Object} params
+ * @param {string} params.name
+ * @param {string?} params.width
+ * @param {((val: string) => boolean)?} params.isValid A function that is called with
+ * the value of the input and expects boolean to be returned representing the
+ * validity of the input.
+ * @returns {React.Element}
+ */
+const FormikComboBox = ({
+  name,
+  width,
+  isValid = () => true,
```

Comment:
Why is this needed? If the Yup validation fails then this would show an error anyways.